File: week 2/proplem set/mario-less/mario.c
#include <cs50.h>
#include <stdio.h>
int main(void){

    int num;
    do{
        num = get_int("Height: ");
    }while(num < 1 || num > 8);

    for(int i = 1; i <= num; i++){
        for(int j = 1; j <= num - i; j++){
            printf(" ");
        }

        for(int j = 1; j <= i; j++){
            printf("#");
        }
        printf("\n");
    }
}
// mkdir

File: week 7/mario.py
"""

from cs50 import get_int

while(True):
    n = get_int('height ')

    if n > 0:
        break

for i in range(n):
    print('#')

"""

"""

for i in range(3):
    print('?',end='')
print()

"""


for i in range(3):
    for i in range(3):
        print('#',end='')
    print()
